10.0.0 Dragonflight hotfix

AngeliqueDawnbringer opened the pull request:
I created a quick hotfix by removing the setGradient parts as it "doesn't work and there are some items renamed. I only pointed them to the native UI elements, not created or reintroduced new ones.

AngeliqueDawnbringer committed:
10.0.0 Dragonflight hotfix

QuinnyNZ commented:
OMG. Thank you so much. Love this addon. That anyone is working on it is incredible. Now to figure out how/what to copy to get working in game.

AngeliqueDawnbringer commented:
Until Frank pushes it, you could just download the entire zip from my master branch.

https://github.com/AngeliqueDawnbringer/PerfectRaid

QuinnyNZ commented:
Can't get to work (added the 4 changed files) so keeps waiting and hoping :( Very nub at this. Hopes accepted soon and a zip up

badamba commented:
Hello @AngeliqueDawnbringer - I can't find any other place to communicate about this and am wondering whether by chance you're checking out 11.0 compatibility? I limped along with PerfectRaid by modding the config file by hand as the options editor was broken, but it's completely dead since the war within pre-patch.  The addon's been really helpful to reduce cognitive load.

AngeliqueDawnbringer committed:
Create README.md

AngeliqueDawnbringer commented:
I have looked at it and I did have a somewhat working version at the day of launch/beta. They however changed the way buff works and some of the healing communication changed. I am currently swamped with work and for the time being I have recreated a similar layout in "Cell" by enderneko and wrote some code how to make it look like PerfectRaid. I love perfect raid as it reduced the information overload and it was simple and clean. Don't expect an update any time soon :(

badamba commented:
Would you be willing to share your Cell config? That's also what I did since it comes the closest, but it has some features missing like text indicators for buffs and debuffs which I added a feature request for.

AngeliqueDawnbringer commented:
When I have fixed the bugs on the layouts I created, I will fork it for you.
Unfortunately, I currently am swamped with work (positively) and my focus between 9-19 is pretty much that at the moment while getting ready for raid next week, but since I want to be raid ready, I will see how far i'll get.

Basically what I did is I rewrote the code to allow for negative values on the icons and a single column of info and move the names with a negative parameter and alight to right. I am currently working on changing the ICON part to be name or ICON or custom name.

What I am trying to fix is making it a addon to cell called "PerfectCell" 😉 see what I did there 😉`?

________________________________
From: badamba ***@***.***>
Sent: Thursday, September 5, 2024 07:48
To: frankdekker/PerfectRaid ***@***.***>
Cc: Angelique Dawnbringer ***@***.***>; Mention ***@***.***>
Subject: Re: [frankdekker/PerfectRaid] 10.0.0 Dragonflight hotfix (PR #6)


Would you be willing to share your Cell config? That's also what I did since it comes the closest, but it has some features missing like text indicators for buffs and debuffs which I added a feature request for.

—
Reply to this email directly, view it on GitHub<https://github.com/frankdekker/PerfectRaid/pull/6#issuecomment-2330653458>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB36TETG5HBKTV3KGXCRCITZU7WCNAVCNFSM6AAAAABNTOFSCOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZQGY2TGNBVHA>.
You are receiving this because you were mentioned.

badamba commented:
Thank you! I appreciate it. I also code... though have no experience with LUA or coding WoW addons but have been taking a look. I tried tweaking https://github.com/enderneko/PerfectRaid to get it to work but no luck yet... 

What do you mean by negative values on icons? Having the buff/debuff icon be a name or text would be super helpful... I know people who have disabilities and they are really suffering with PerfectRaid not working anymore, since they depend on that visually simpler representation of data for unitframes, so I'm trying to help.
